SnowMaker.IntegrationTests/SnowMaker.IntegrationTests.csproj

```diff
@@ -5,13 +5,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="9.4.0" />

```

Comment:
Thanks. Sorry I didn't use your PR, as it also required some using directives to be added.